Network/network-exchange-support-guide.md

```diff
@@ -0,0 +1,30 @@
+{{{
+  "title": "Network Exchange Support Guide",
+  "date": "06-22-2018",
+  "author": "Jason Holland",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Support Options
+
+Network Exchange support includes phone, email, chat and support tickets. Details on these options are available at *[How do I get support for CenturyLink Cloud services?](../Support/how-do-i-report-a-support-issue.md)*
+
+### Using the Support Ticketing Web Interface
+
+The following Knowledge Base articles explain how to access and use the Support ticketing web interface 
+* *[Accessing the Support ticketing web interface](../Support/zendesk-login-help-for-helpdesk-ticketing-and-kb-access.md)*
+* *[Using the CenturyLink Cloud Support web interface](../Support/using-the-help-desk-web-ui.md)*
+
+### Support Ticket Prioritization and Escalation
+
+The Network Exchange support ticket prioritization and response time details are available at *[Ticket Prioritization Matrix](../Support/ticket-prioritization-matrix.md)*.
+
+The process to escalate support tickets can be found at *[How do I escalate a ticket?](../Support/How-do-i-escalate-a-ticket.md)*
```

Comment:
Change
(../Support/How-do-i-escalate-a-ticket.md)
to (../Support/how-do-i-escalate-a-ticket.md)  - lowercase h